src/api/svg.ts

```diff
@@ -835,6 +847,25 @@ const parseAttributes = (
 
         if (!params) return letter || '';
         switch (letter?.toLocaleLowerCase()) {
+          case 'm': 
+          case 'l': {
+            const groupedParams = groupBy<string>(params, 2);
+            return groupedParams
+              .map((p, pIndex) => {
+                const [x, y] = p;
+                const xReal = parseFloatValue(x, inherited.width) || 0;
+                const yReal = parseFloatValue(y, innerHeight) || 0;
+                if (letter === letter.toLowerCase()) {
+                  const { width: dx, height: dy } = newConverter.size(xReal, yReal);
+                  return (pIndex > 0 || letter === 'l' ? 'l' : 'm') + [dx, -dy].join(',');
```

Comment:
Can you explain the condition on pIndex?

Reply:
It's so this case gets taken care of:

![image](https://user-images.githubusercontent.com/54117007/199791327-c1370c82-bd7b-4906-bad4-35870c2dc980.png)

I thought about making a case for each, but it seemed redundant when the differentiation can be done in one line. I could add a comment for clarity, if you think it's needed.